flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "automerge-flathubbot-prs": true
```

Comment:
I would advice against this.

Reply:
Except I have seen packages break more often than not with autogenerated updates. Proton-bridge for example breaks around 1/5 of the updates.